packetbeat/protos/tls/tls.go

```diff
@@ -260,6 +260,8 @@ func (plugin *tlsPlugin) createEvent(conn *tlsConnectionData) beat.Event {
 	if server.parser.hello != nil {
 		serverHello = server.parser.hello
 		tls["server_hello"] = serverHello.toMap()
+	} else {
+		serverHello = emptyHello
```

Comment:
We could init serverHello and clientHello with emptyHello on line 247. Would probably make the a bit more readable (for me).